kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -319,7 +320,7 @@ var LoginView = BaseView.extend({
         // has finished loading.
         this.set_login_button_state();
         this.$(".password-btn").removeAttr("disabled");
-        this.$("#id_username").on( "autocompletesearch", this.set_login_button_state);
+        this.$("#id_username").on(this.set_login_button_state);
```

Comment:
Hi @benjaoming I will try to fix that one.
Yeah your right about that comment. If I remove that `event`. It causing an error.

I will try to figure out to fix and fix the issue.
Thanks for the tip.